### ISSUE TITLE
plotly graph

### ISSUE THREAD

**Issue by pandya6988** [gradio-app/gradio#605]
**outputs="plot" doesn't work for plotly plots.**

**Example code:**
my function return plotly plot as bellow:

```
import plotly.express as plx
def my_func(some_data):
    x = some_data['x']
    y = some_data['y']
    return fig = plx.scatter(df, x='x', y='y', color=label, hover_data=['text'])

```

`interface = gr.Interface(get_plot, inputs='some data', outputs='plot')`

Cannot plot this.

**Comment by pandya6988** [comment]
```
def get_plot(model_name, chunk, version='training', label='senti'):
    try:
        with open(f'data/{model_name}', 'rb') as f:
            model_dict = pickle.load(f)
    except: 
        error_msg = bc.bcolors.FAIL + 'Unvalid file.'
        return error_msg 
    
    pca2 = PCA(n_components=2)
    pca2_chunk = pca2.fit_transform(model_dict[version]['encoded_text'][chunk])
    df = pd.DataFrame()
    df['x'] = pca2_chunk[:,0].squeeze().tolist()
    df['y'] = pca2_chunk[:,1].squeeze().tolist()
    df['language'] = model_dict[version]['language']
    df['senti'] = model_dict[version]['sentiment']
    df['text'] = [text[:250]for text in model_dict[version]['text']]

    fig = plx.scatter(df, x='x', y='y', color=label, title=f'PCA {model_name} {version} {chunk}', hover_data=['text'], opacity=0.2)
    
    print(bc.bcolors.HEADER + "Converting to html ..." + bc.bcolors.ENDC)
    fig.write_html("test1.html", include_plotlyjs='cdn')
    f = codecs.open("test1.html",'r','utf-8')
    doc = BeautifulSoup(f)
    print(bc.bcolors.HEADER + "Done..." + bc.bcolors.ENDC)
    return str(doc)
```

This is not working as well.

**Comment by abidlabs** [comment]
Thanks @pandya6988 we currently don't support plotly graphs, but we are working on adding it!

**Comment by abidlabs** [comment]
Closed via #805

**Comment by abidlabs** [comment]
Plotly is now supported with the new `Plot` component as of version 2.9.3!
